definitions/divorce/json/CaseEventToFields/CaseEventToFields-pet-sol-selects-own-org-prod.json

```diff
@@ -1,218 +0,0 @@
-[
-    {
```

Comment:
moved to https://github.com/hmcts/div-ccd-definitions/pull/707/files#diff-354b66c7314e173d7aaf99a8a7f511faaa769915f4f6b9d05d4521db3242a8f2R2013-R2228

This will be released together as a RRJ.